src/llm_utils.py

```diff
@@ -5,178 +5,136 @@
 from google import genai
 from typing import List, Dict
 
+from github_utils import get_valid_hunks
+
 def analyze_code_changes(diff_content: str) -> List[Dict]:
     """
     Analyze code changes using OpenAI's GPT model
     Returns a list of review comments
     """
-    API_KEY = os.getenv('GEMINI_API_KEY')
 
```

Comment:
The old prompt structure is replaced.
```suggestion
|
```
N/A